### DISCUSSION TITLE
Global system keyboard input monitoring?

### DISCUSSION THREAD

**Discussion by mattrobin** [bevyengine/bevy#6961] [Q&A] [answered]
Hello! Does Bevy's input handling have a way to monitor keyboard input even when the application is not in focus? Or will I need a third party library to handle this? I can't seem to find anything in the documentation that suggests it's available, but I thought I should ask in case I'm missing something. Thank you for your time!

**Comment by alice-i-cecile** [comment, marked as answer]
As far as I'm aware winit does not provide this, in part because this only works on some platform.

You'll need an external library.

**Comment by rparrett** [comment]
https://github.com/laundmo/bevy_global_input may work for you.

**Reply by mattrobin** [reply]
> https://github.com/laundmo/bevy_global_input may work for you.

Thanks, but, unfortunately, it looks like this doesn't work for macOS, which I want to support. (Seemly due to dependency `mki` not supporting macOS.)
